tests/test-http-client-transport/src/test/java/org/eclipse/jetty/http/client/AbstractTest.java

```diff
@@ -206,6 +206,7 @@ protected void startClient() throws Exception
         clientThreads.setName("client");
         clientThreads.setDetailedDump(true);
         client = newHttpClient(provideClientTransport(transport), sslContextFactory);
+        client.setMaxConnectionsPerDestination(Integer.MAX_VALUE);
```

Comment:
This allocates memory.
Set a more sane value here please.

Reply:
Ah that would explain my out of memory exceptions!!! thank!